src/platforms/go/common/migration.mdx

```diff
@@ -386,10 +386,14 @@ sentry-go
 go get github.com/getsentry/sentry-go/http
 ```
 
+<Break />
+
 ```go
 import sentryhttp "github.com/getsentry/sentry-go/http"
 ```
 
+<Break />
```

Comment:
An alternative that might be more clear is to set the full examples, by duplicating the `import` statement and having one of the options in a different tab. I don't know which option is more clear and easier to understand, so your call.

Reply:
I still consider the migration guide needs a rewrite to be more approachable.

Instead of a big bang change, I decided to do the easy part(s) first.

In this particular part, I don't think multiple tabs help, because they'd make it a bit harder to see old-new side by side.

And the tab grouping we're fixing here was a case that was never intended to be grouped.